weather_dl/download_pipeline/pipeline.py

```diff
@@ -150,7 +168,32 @@ def skip_partition(config: t.Dict, store: Store) -> bool:
     return False
 
 
-def prepare_partitions(config: t.Dict, store: t.Optional[Store] = None) -> t.Iterator[t.Tuple]:
+def get_subsections(config: t.Dict) -> t.List[t.Dict]:
+    """Collect parameter subsections from main configuration.
+
+    If the `parameters` section contains subsections (e.g. '[parameters.1]',
+    '[parameters.2]'), collect a list of the subsection key-value
+    pairs. Otherwise, return a list of an empty dictionary (i.e. to address
+    the case where there are no subsections).
+
+    This is useful for specifying multiple API keys for your configuration.
+    For example:
+    ```
+      [parameters.deepmind]
```

Comment:
Maybe use more generic names than team names ?

Reply:
I've only copied the old docstring message from one place to another here. Further, I like these names. I think they add  color to how users can think about licenses and who they belong to, using course examples from Alphabet.